bin/init.bash

```diff
@@ -6,6 +6,7 @@
 # It's not to be executed on its own but rather via `ck8s init`.
 
 : "${CK8S_CLUSTER:?Missing CK8S_CLUSTER}"
+: "${CLUSTER_PROVISIONER:?Missing CLUSTER_PROVISIONER}"
```

Comment:
```suggestion
```
We will use `CK8S_CLUSTER_PROVISIONER` instead, which is loaded in common (if it is set).